posthog/api/property_definition.py

```diff
@@ -48,8 +68,13 @@ def get_queryset(self):
                     names = tuple(properties_to_filter.split(","))
                     name_filter = f"AND name IN %(names)s"
                 else:
-                    name_filter = ""
+                    search = self.request.GET.get("search", None)
                     names = ()
+                    name_filter = (
+                        ""
+                        if search is None
+                        else f"AND ngramSearchCaseInsensitiveUTF8(name, {search}) > {self.search_threshold}"
```

Comment:
Tangential to this PR but maybe we could convert `ee_property_definitions` query to Django ORM and use the same `FuzzySearchFilterBackend` to avoid code with duplicated behavior which could lead to an inconsistent experience.

Reply:
Actually after second thought, this is not going to work. `ngramSearchCaseInsensitiveUTF8` is Clickhouse but the EE model for EventDefinition is on Postgres too. Definitely worth adding tests for EE too.

Reply:
@paolodamico Hmmm... experimentally I got this to work, and the idea here is that we use Django's wrapper for `pg_trgm` on Postgres, and `ngramSearchCaseInsensitiveUTF8` (which is a trigram search, too) on CH. The results should be nearly identical as it is actually a pretty simple algorithm.

Because it's wrapped up in the else block for importing EE models, `from ee.models.property_definition import EnterprisePropertyDefinition` it should only run on EE, that is Clickhouse, correct? Or is it possible to have EE without Clickhouse and cause this query to run on Postgres?

Reply:
We're just querying postgres here.

Reply:
Yeah the thing is that even though`EnterprisePropertyDefinition` is on EE, it's a Postgres model, so `ngramSearchCaseInsensitiveUTF8` won't work.